.vib/thanos/bucketweb/cypress/cypress/integration/thanos_bucketweb_spec.js

```diff
@@ -0,0 +1,21 @@
+/// <reference types="cypress" />
+
+it('allows to see runtime & build information', () => {
+    cy.visit('/');
+    cy.contains('.dropdown-toggle', 'Status').click();
+    cy.contains('[class="dropdown-item"]', 'Runtime & Build Information').click();
+    cy.contains('Runtime Information').should('be.visible');
+    cy.get('h2').contains('Build Information').should('be.visible');
```

Comment:
```suggestion
    cy.get('h2').contains('Build Information');
```

Do you need this `should('be.visible');` assertion here?